.github/workflows/publish-release.yml

```diff
@@ -39,6 +39,8 @@ jobs:
       - name: Publish and close repository
         run: |
           ./gradlew --no-daemon --stacktrace --max-workers=1 publish closeAndReleaseStagingRepository
+          -PbuildJdkVersion=17 \
+          -PtestJavaVersion=17 \
```

Comment:
We don't need `PtestJavaVersion` if it's the same as `PbuildJdkVersion`.
And because we set up JDK 17, I think we don't need both `PbuildJdkVersion` and `PtestJavaVersion`.